workflow/pred_temperature_autoencoder.ipynb

```diff
@@ -0,0 +1,1298 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # check this doc if not https://era5cli.readthedocs.io/en/latest/instructions.html#register-at-copernicus-climate-data-service
</pre><p>https://era5cli.readthedocs.io/en/stable/getting_started/</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Updated!</p>